components/rowListScene.brs

```diff
@@ -21,3 +34,32 @@ sub init()
         m.rowList.itemSize = [1400, INT(m.rowList.itemSize[1] * m.global.ratio)]
     end if
 end sub
+
+sub animateScroll(direction as String)
+    if direction = "down" and m.rowFocused < 5
+        m.rowFocused++
+        for i = 0 to 5
+            m.scrollAnimation.getChild(i).keyValue = [[0, m.parent.getChild(i).translation[1]], [0, m.parent.getChild(i).translation[1] - 300]]
+        end for
+        m.parent.getChild(m.rowFocused).setFocus(true)
+        m.scrollAnimation.control = "start"
+    else if direction = "up" and m.rowFocused > 0
+        for i = 0 to 5
+            m.scrollAnimation.getChild(i).keyValue = [[0, m.parent.getChild(i).translation[1]], [0, m.parent.getChild(i).translation[1] + 300]]
+        end for
+        m.parent.getChild(m.rowFocused).setFocus(true)
+        m.scrollAnimation.control = "start"
+    end if
+end sub
```

Comment:
Instead of doing this put all `RowList`s in a `Group` and scroll the `Group`.

Reply:
Not done

Reply:
Done